packages/gdl-frontend/pages/books/_book.js

```diff
@@ -244,7 +244,13 @@ class BookPage extends React.Component<{ book: Book }> {
                     </Typography>
 
                     <Typography paragraph variant="subtitle1">
-                      <Trans>from {book.publisher.name}</Trans>
+                      <FormattedMessage
+                        id="from name"
```

Comment:
Denne kan formatteres til å oversette `from` bare og ikke `from {name}`

Reply:
Den fikk den `id'en` for å unngå duplikat i `id´. Men vi kan se hva vi gjør om vi bare ønsker å bruke `id`